src/Main.java

```diff
@@ -1,8 +1,117 @@
 public class Main {
     public static void main(String[] args) {
 
-        System.out.println("Hello world!");
+        //task 1
+        System.out.println("task 1");
 
-        //initial commit
+        byte age = 18;
+        if (age >= 18) {
+            System.out.println("Если возраст человека " + age + ", то человек совершеннолетний");
+        }
+        else {
+            System.out.println("Если возраст человека " + age + ", то возраст совершеннолетия еще не наступил, нужно немного подождать");
+            //System.out.println("КожАнный еще на личиночной стадии");
+        }
+
+        //task 2
+        System.out.println("task 2");
+
+        byte temperature = 12;
+        if (temperature < 5) {
+            System.out.println("На улице холожно, нужно надель шапку");
+        }
+        else {
+            System.out.println("Сегодня тепло, можно идти без шапки");
+        }
+
+        //task 3
+        System.out.println("task 3");
+
+        byte speed = 99;
+        if (speed > 60) {
+            System.out.println("Ты заплатишь за это! (штраф)");
+        }
+        if (speed <= 60){
+            System.out.println("Можно ехать спокойно");
+        }
+
+        //task 4
+        System.out.println("task 4");
+
+        byte ageInTask4 = 99;
```

Comment:
зачем создавать новую переменную, если можно просто переиспользовать уже созданную выше?
```suggestion
        byte ageInTask4 = 99;
```
```suggestion
        age = 99;
```